### DISCUSSION TITLE
Add on not being displayed on add-ons list after clicking install.

### DISCUSSION THREAD

**Discussion by briancunning** [SBCV/Blender-Addon-Photogrammetry-Importer#54] [Q&A]
When I cliick install I see in the console that the correct folder has been added to the add-ons folder. However the add-on does not appear on the list of installed add-ons.
I restart Blender and still it does not appear on the list. 
I have tried blender 3.4, 3.3, 3.2 and 3.1 and they all have the same issue.

**Comment by SBCV** [comment]
Sorry for the late reply. Does the problem appear with the current version as well?

**Comment by HerrFile** [comment]
Same problem here. Tried installing by  drag&drop or via UI. Started blender as an admin.
Files are copied to the scripts folder but nothing showing up in Blender. Tried 3.0 and 5.0

**Comment by SBCV** [comment]
Check out the latest release from today:
https://github.com/SBCV/Blender-Addon-Photogrammetry-Importer/releases/tag/v2026.02.16

This should work for Blender 5
